### ISSUE TITLE
run server.py issues

### ISSUE THREAD

**Issue by joamatab** [dougthor42/trendlines#175]
the docker container version works, 

however the version of the repo has some issues, are you sure the version on the requirements.txt are correct?
 
```
python3 -m venv venv   
source activate/venv/bin/activate.fish
pip install -r requirements.txt
pip install -e .
python run_server
```

(venv) ⋊> ~/c/d/trendlines on master ⨯ python runserver.py                                                                                                                                                                                                                                                                                                           16:16:18
Traceback (most recent call last):
  File "runserver.py", line 7, in <module>
    import trendlines.orm as orm
  File "/Users/joaquin/files/code-github/dashboards/trendlines/src/trendlines/__init__.py", line 4, in <module>
    from trendlines.app_factory import create_app
  File "/Users/joaquin/files/code-github/dashboards/trendlines/src/trendlines/app_factory.py", line 11, in <module>
    from trendlines import routes
  File "/Users/joaquin/files/code-github/dashboards/trendlines/src/trendlines/routes.py", line 8, in <module>
    from marshmallow_peewee import ModelSchema
  File "/Users/joaquin/files/code-github/dashboards/trendlines/venv/lib/python3.7/site-packages/marshmallow_peewee/__init__.py", line 9, in <module>
    from .schema import ModelSchema # noqa
  File "/Users/joaquin/files/code-github/dashboards/trendlines/venv/lib/python3.7/site-packages/marshmallow_peewee/schema.py", line 4, in <module>
    from marshmallow.compat import with_metaclass
ModuleNotFoundError: No module named 'marshmallow.compat'


and pytest has the same issue

E   ModuleNotFoundError: No module named 'marshmallow.compat'

**Comment by dougthor42** [comment]
Hmm... interesting! Indeed, it looks like I'm missing a pin for `marshmallow` or perhaps `webargs`.

Try installing `marshmallow==2.20.5` as a temporary workaround.

I'll be able to take a more in-depth look at this in about 16hrs.

**Comment by dougthor42** [comment]
Yeah, looks like the root cause was that I was relying on `marshmallow-peewee` to install `marshmallow`, and they only have the requirement `marshmallow >=  2.15.0`.

I've merged #176 which should resolve this issue. Pull `master` and then run

```
pip install -r requirements.txt
```

again and you should be all set.

**Comment by joamatab** [comment]
work great! thank you!
